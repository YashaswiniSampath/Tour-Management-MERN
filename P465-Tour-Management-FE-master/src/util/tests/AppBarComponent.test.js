import renderer from 'react-test-renderer';
import LoginNavComponent from '../LoginNavComponent';
import { BrowserRouter as Router } from 'react-router-dom';

it('renders page', () => {
    const component = renderer.create(
        <Router><LoginNavComponent /></Router>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})