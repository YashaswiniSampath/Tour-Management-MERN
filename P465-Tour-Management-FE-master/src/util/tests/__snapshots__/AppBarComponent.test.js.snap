// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-11b3ww9-MuiPaper-root-MuiAppBar-root"
  style={
    Object {
      "backgroundColor": "#2484BF",
      "width": "100%",
    }
  }
>
  <div
    className="MuiToolbar-root MuiToolbar-regular css-1l9c7aa-MuiToolbar-root"
  >
    <img
      alt="EZTravel Logo"
      className="MuiBox-root css-3jnihy"
      src="EZTravelLogo.png"
    />
    <a
      className="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-m5x0z-MuiTypography-root"
      href="/"
    >
      EZTravel
    </a>
    <div
      className="MuiBox-root css-i9gxme"
    />
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textInherit MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorInherit MuiButton-root MuiButton-text MuiButton-textInherit MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorInherit css-1uh6h56-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      Return to Home
    </button>
  </div>
</header>
`;
